### PR TITLE
Add support for LDO Picobilical

### PR DESCRIPTION
**Pull request by ndanyluk**
Ok, so I suck at git and accidentally screwed up my repo, nuking #208 (and my V0's config) in the process.

I've gone ahead and re-created the config, using the LDO Nitehawk as a reference point.  There are some minor changes from the last known state of #208 (I removed the temp sensor changes and made them local to the config like the Nitehawk has), but the config is mostly identical.

### CODE REVIEW

File: user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    PART_FAN=MCU_PCF , E_FAN=MCU_HEF ,
+
+    PB_PCB_TEMP=MCU_CT ,
+
+    # If you want to use both of the independent neopixel ports,
+    # comment the below line and uncomment the left/right neopixel line
+    # NOTE: YOU WILL NOT BE ABLE TO USE THE STOCK KLIPPAIN
+    # MACROS WITH THE DUAL NEOPIXEL CONFIGURATION!
+    LIGHT_NEOPIXEL=MCU_NEOPIXEL_1
+    # RIGHT_NEOPIXEL=MCU_NEOPIXEL_1 , LEFT_NEOPIXEL=MCU_NEOPIXEL_2
```
Comment (Frix-x) [inline user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg:34]: Can you rename the second one `LIGHT_NEOPIXEL_2` and keep the original `LIGHT_NEOPIXEL` as the first one?

This way this second pin can be used without touching the original Klippain pin and avoid breaking compatibility with the original macros

File: user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## and having the X endstop plugged to the toolhead MCU
+# [stepper_x]
+# endstop_pin: ^toolhead:X_STOP
+
+[temperature_sensor picobilical_temp]
+sensor_type: CMFB103F3950FANT
+sensor_pin: PB_PCB_TEMP
+pullup_resistor: 2200
+min_temp: 0
+max_temp: 100
```
Comment (Frix-x) [inline user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg:65]: Can you add two blank new lines at the end of the file to allow it to be injected to the mcu.cfg user file easily?

**Review by Frix-x — CHANGES_REQUESTED** [review]
Thanks, I think I'm almost ready to merge it. Just a slight change regarding the neopixel pins to avoid breaking compatibility with the other original Klippain files

File: user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg (hunk 1 above)
Comment (ndanyluk) [reply user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg:34]: whoops, I totally missed this sorry! Will fix now

File: user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg (hunk 2 above)
Comment (ndanyluk) [reply user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg]: Done, closing

**Comment by ndanyluk** [comment]
Hey @Frix-x I think this should do it - for some reason Github doesn't want to display the blank 2 lines at the end of the file - is that normal?

**Review by Frix-x — APPROVED** [review]

**Comment by Frix-x** [comment]
Thanks, it's ok for me! I'll merge it :)

**Pull request merged**